lms/services/jstor.py

```diff
@@ -2,27 +2,43 @@
 
 
 class JSTORService:
-    def __init__(self, settings):
-        self._settings = settings
+    def __init__(self, enabled, site_code):
+        """
+        Initialise the JSTOR service.
+
+        :param enabled: Whether JSTOR is enabled on this instance
+        :param site_code: The site code to use to identify the organization
+        """
+        self._enabled = enabled
+        self._site_code = site_code
 
     @property
-    def enabled(self):
-        return self._settings.get("enabled", False)
+    def enabled(self) -> bool:
```

Comment:
Should we  call this from:

https://github.com/hypothesis/lms/blob/jstor-save-site-code/lms/resources/_js_config/file_picker_config.py#L108

Reply:
Yes. Yes we should